plugins/chat/assets/javascripts/discourse/components/chat-drawer/header/right-actions.js

```diff
@@ -3,5 +3,10 @@ import { inject as service } from "@ember/service";
 
 export default class ChatDrawerHeaderRightActions extends Component {
   @service chat;
+  @service router;
```

Comment:
where is this used?

Reply:
ah yeah not needed for now, i was going to do some route based changes for the drawer here but saw it would be more complex so left it for now